system/puppet.py

```diff
@@ -77,6 +77,11 @@
   certname:
     description:
       - The name to use when handling certificates.
+    version_added: "2.1"
```

Comment:
not so fast :) you cut off the 
~~~yaml
required: false
default: None
~~~

from the previous argument. Could you add it again?